tasks/main.yml

```
@@ -93,6 +93,20 @@
   args:
     chdir: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}"
 
+# site-install.
+- name: check if site-install config option is enabled.
+  set_fact:
+    deploydrupal_site_install_config_option: "{{ '--existing-config' if deploydrupal_site_install_config == true else '' }}"
```

I didn't know this would work (I thought to use `if` / `else`, we had to use `{%` / `%}`). In any case, this could also look like this:

```suggestion
    deploydrupal_site_install_config_option: "{{ deploydrupal_site_install_config | ternary('--existing-config', '') }}"
```